zeppelin-zengine/src/main/java/org/apache/zeppelin/notebook/Note.java

```diff
@@ -154,6 +155,12 @@ public void setName(String name) {
     this.name = name;
   }
 
+
+  public boolean isPersonalizedMode() {
```

Comment:
so basically it means that `personalized mode` depends on the `config` field of the `note` itself, and the good part is that we have fine grained switch per note, although can't see where we'll be switching it. i guess it's left for frontend work